Templates/RemoteRepo/template.json

```
@@ -0,0 +1,19 @@
+{
+    "template_name": "RemoteRepo",
+    "origin": "The primary repo for RemoteRepo goes here: i.e. http://www.mydomain.com",
```

`template.json` files do not have an origin field.
```suggestion
    "origin": "https://github.com/o3de/o3de",
```